client/src/component/DOS/DefineAudit/SelectContestsForm.tsx

```diff
@@ -207,8 +219,9 @@ class SelectContestsForm extends React.Component<FormProps, FormState> {
 
         };
         const filteredData = _.filter(sortedData, filterFunc);
+        const uniqData = _.filter(filteredData, uniqByName);
```

Comment:
Is there a way to use `_.uniqBy` here instead?

Reply:
I tried but I couldn't get the types to agree. Suggestions welcome.